src/main/resources/locales/cy/template.json

```diff
@@ -0,0 +1,18 @@
+{
+  "serviceNameText": "Court and tribunal hearings",

```

Comment:
didnt realise i could read welsh

Reply:
yeah, we don't have all translations. I've been only provided with one sentence, they want to see the switch functionality rather than all translations